quizz.md

```diff
@@ -0,0 +1,213 @@
+## Ruby Basics & Enumerables (meets Beauty and the Beast)
+
+
+### Question 1
+
+Define a method called `offerRose`, which should take one argument named person.
+
+When called, the method should `puts`, "Would you take this rose and help out
+an old beggar, X?", where X is the person passed into the method.
+
+Demonstrate calling the method with an argument of "young prince".
+
+Write your code here:
+```
+def offerRose (person)
+  puts "Would you take this rose and help out an old beggar, " + person + "?"
+end
```

Comment:
:+1: This is correct.  Many rubyists would use Ruby's string interpolation (`#(person}`) to inject the variable into the string.  Also, we rarely put a space between the method name and the argument parens.